drivers/watchdog/Kconfig.dw

```diff
@@ -9,3 +9,12 @@ config WDT_DW
 	depends on DT_HAS_SNPS_DESIGNWARE_WATCHDOG_ENABLED
 	help
 	  Synopsys DesignWare Watchdog driver.
+
+if WDT_DW
+
+config WDT_DW_PAUSE_BY_DBG
```

Comment:
I can't see any location in the driver where the pause by debug functionality is enabled depending on this config being enabled. Am I missing something?

Reply:
@martinjaeger it is present in wdt_dw_common.c

@softwarecki True, software configuration doesn't have any effect with respect to configuring synopsis dw IP.

Even if the platform design is such that it is configured to pause the watchdog, user will assume that he needs to configure that as a part of wdt_setup function, because it is defined in watchdog.h(user interface file).

For example user uses sample application and lets say hardware line pause by debug signal is connected to watchdog(which is true in Agilex5 platform). Sample application calls wdt_setup() function with the WDT_OPT_PAUSE_HALTED_BY_DBG. The wdt_dw driver will respond that the WDT_OPT_PAUSE_HALTED_BY_DBG is not supported even though user has sent a correct flag. But this is not true in case of agilex and agilex5 platform
This is the reason I introduced the flag. Let me know your view on this.

Reply:
In that case, it may be enough to modify the error message to inform user that it is not configurable by the software or simply ignore that flag. Personally, I wouldn't add a fake Kconfig option to bypass flag checking.

Reply:
removed this option as this is not configurable through application and added log message to inform the user about the same if user tries to configure this option